subprojects/scala/src/main/java/org/gradle/api/plugins/scala/ScalaBasePlugin.java

```diff
@@ -279,6 +288,7 @@ private void configureScaladoc(final Project project, final ScalaRuntime scalaRu
             scalaDoc.getConventionMapping().map("destinationDir", (Callable<File>) () -> project.getExtensions().getByType(JavaPluginExtension.class).getDocsDir().dir("scaladoc").get().getAsFile());
             scalaDoc.getConventionMapping().map("title", (Callable<String>) () -> project.getExtensions().getByType(ReportingExtension.class).getApiDocTitle());
             scalaDoc.getConventionMapping().map("scalaClasspath", (Callable<FileCollection>) () -> scalaRuntime.inferScalaClasspath(scalaDoc.getClasspath()));
+            scalaDoc.getConventionMapping().map("isScala3", (Callable<Boolean>) () -> isScala3(scalaDoc.getScalaClasspath()));
```

Comment:
As indicated, remove this as the information is obtained in another way

Reply:
Removed